packages/core/src/tools/utils/urlPolyfill.ts

```diff
@@ -77,11 +73,12 @@ export function getLocationOrigin() {
 }
 
 /**
- * IE fallback
- * https://developer.mozilla.org/en-US/docs/Web/API/HTMLHyperlinkElementUtils/origin
+ * Fallback
+ * On IE HTMLAnchorElement origin is not supported: https://developer.mozilla.org/en-US/docs/Web/API/HTMLHyperlinkElementUtils/origin
+ * On Firefox window.location.origin is "null" for file: URIs: https://bugzilla.mozilla.org/show_bug.cgi?id=878297
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about adding a test case for file URI?

Reply:
Hum, there are two issues with Firefox with file URI:
- The URL.origin contain a "null"
- The URL.host is empty

Which is why we have this [check](https://github.com/DataDog/browser-sdk/blob/91ff5786bd99a3ad8059d7f0f677c2ea550351e6/packages/core/src/tools/utils/urlPolyfill.spec.ts#L26) in our tests

My fix was only not to make the normalizeUrl() break on firefox for file URI, however `normalizeUrl("file://foo.bar")` will return
- "file:///" for firefox
- "file://foo.bar" for other browser 

I tried to reflect that in our tests